ImageLounge/src/DkGui/DkCentralWidget.h

```diff
@@ -145,6 +145,7 @@ class DllCoreExport DkCentralWidget : public DkWidget
 
 signals:
     void imageUpdatedSignal(QSharedPointer<DkImageContainerT>) const;
+    void directoryLoadedSignal(QString) const;
```

Comment:
```suggestion
    void directoryLoadedSignal(const QString&) const;
```

When passing objects into functions, we usually pass them by reference to avoid copying.